app/static/js/bonnerManagement.js

```diff
@@ -69,7 +72,30 @@ function addRequirement() {
     addRequirementsRowHandlers()
     newRow.find("input").focus()
 }
-
+function addCohort(){
+    let years = $('#v-pills-tab .nav-link').map((i, element) => {return Number($(element).data('year'))}).get();
+    let newCohortYear = Math.max(...years) + 1;
+    // Remove class 'active' from the currently active tab
+    $('#v-pills-tab .active').removeClass('active');
+    $('#v-pills-tabContent .tab-pane').removeClass('show active')
+    $('#v-pills-tab #addCohort').after(`<button class="nav-link active" id="v-pills-${newCohortYear}-tab" data-bs-toggle="pill" data-bs-target="#v-pills-${newCohortYear}" type="button" role="tab" data-year="${newCohortYear}" aria-controls="v-pills-${newCohortYear}" aria-selected="{{aria}}">${newCohortYear} - ${newCohortYear + 1}</button>`)
```

Comment:
Finding it hard to understand what's going on here, I think adding some comments might be more helpful.

Reply:
Done!